jena-fuseki2/jena-fuseki-core/src/main/java/org/apache/jena/fuseki/server/Dispatcher.java

```diff
@@ -463,6 +509,8 @@ private static Operation gspOperation(HttpAction action, HttpServletRequest requ
      * (POST, PUT, DELETE, PATCH)
      */
     private static boolean isReadMethod(HttpServletRequest request) {
+        // OPTIONS needs to knwo if the EndPoitn set contains GSP_R or GSP_RW.
```

Comment:
s/knwo/know and s/EndPoitn/Endpoint or EndPoint

Reply:
Removed comment (it's not right anymore).